Backend/app/provider-platform/dynamic-offer-driver-app/Main/spec/Storage/configs.yaml

```diff
@@ -139,6 +140,36 @@ ValueAddNP:
       kvFunction: findAllWithKV
       where: enabled
 
+CancellationAndNoShowChargeConfigs:
+  tableName: cancellation_and_now_show_charge_configs
+
+  fields:
+    id : Id CancellationAndNoShowChargeConfigs
```

Comment:
Why do we want it to separate then fareProduct? We can link it through FareProduct, for example if other dimension are coming e.g. peakHours, specialLocation it could be handled automatically.

Here id could be FarePolicyId and FarePolicyId could be the primaryKey. To fetch this, you just first fetch particular fareProduct and then cancellationChargeConfigs. You can call this table CancellationFarePolicy, instead of naming it `CancellationAndNoShowChargeConfigs`

Reply:
Made changes